app/config/events.php

```diff
@@ -97,6 +97,46 @@
         'model' => Response::MODEL_ANY,
         'note' => '',
     ],
+    'functions.create' => [
+        'description' => 'This event triggers when a function is created.',
+        'model' => Response::MODEL_FUNCTION,
+        'note' => 'version >= 0.7',
+    ],
+    'functions.update' => [
+        'description' => 'This event triggers when a function is updated.',
+        'model' => Response::MODEL_FUNCTION,
+        'note' => 'version >= 0.7',
+    ],
+    'functions.delete' => [
+        'description' => 'This event triggers when a function is deleted.',
+        'model' => Response::MODEL_ANY,
+        'note' => 'version >= 0.7',
+    ],
+    'functions.tags.create' => [
+        'description' => 'This event triggers when a function tag is created.',
+        'model' => Response::MODEL_TAG,
+        'note' => 'version >= 0.7',
+    ],
+    'functions.tags.update' => [
+        'description' => 'This event triggers when a function tag is updated.',
+        'model' => Response::MODEL_FUNCTION,
+        'note' => 'version >= 0.7',
+    ],
+    'functions.tags.delete' => [
+        'description' => 'This event triggers when a function tag is deleted.',
+        'model' => Response::MODEL_ANY,
+        'note' => 'version >= 0.7',
+    ],
+    'functions.executions.create' => [
+        'description' => 'This event triggers when a function execution is created.',
```

Comment:
Does the event only trigger when executed over the HTTP endpoint or also on schedule/event?

If only HTTP, I would update this section. Also we are sending update events to any execution, but only send events to specific execution creation? 🤔

Reply:
Okay, just found the part where its being taken care of 👍🏻